Failed CLI test

============================= test session starts ==============================
platform darwin -- Python 3.8.5, pytest-6.1.0, py-1.9.0, pluggy-0.13.1
rootdir: $SRC_DIR
plugins: cov-2.10.1
collected 1 item

tests/test_cli.py F                                                      [100%]

=================================== FAILURES ===================================
______________________________ test_cli_template _______________________________

    def test_cli_template():
>       assert cli.cli() is None

tests/test_cli.py:4: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.8/site-packages/pieparty/cli.py:16: in cli
    args = p.parse_args(args)
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.8/argparse.py:1771: in parse_args
    self.error(msg % ' '.join(argv))
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.8/argparse.py:2521: in error
    self.exit(2, _('%(prog)s: error: %(message)s\n') % args)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = ArgumentParser(prog='pytest', usage=None, description='scRNA visualization tool', formatter_class=<class 'argparse.HelpFormatter'>, conflict_handler='resolve', add_help=True)
status = 2, message = 'pytest: error: unrecognized arguments: tests\n'

    def exit(self, status=0, message=None):
        if message:
            self._print_message(message, _sys.stderr)
>       _sys.exit(status)
E       SystemExit: 2

../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.8/argparse.py:2508: SystemExit
----------------------------- Captured stderr call -----------------------------
usage: pytest [-h] [-V]
pytest: error: unrecognized arguments: tests
=========================== short test summary info ============================
FAILED tests/test_cli.py::test_cli_template - SystemExit: 2
============================== 1 failed in 0.17s ===============================

--- a/{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/cli.py
+++ b/{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/cli.py
@@ -13,7 +13,7 @@ def cli(args=None):
         version="{{ cookiecutter['package_name']}} %s" % __version__,
     )
 
-    args = p.parse_args(args)
+    args, unknown = p.parse_known_args(args)
 
     # do something with the args
     print("CLI template - fix me up!")

